Mobile Buy SDK/Mobile Buy SDK/Operations/BUYCheckoutOperation.m

```diff
@@ -29,15 +29,15 @@
 #import "BUYClient+Internal.h"
 #import "BUYPaymentToken.h"
 #import "BUYRequestOperation.h"
+#import "BUYStatusOperation.h"
 
 @interface BUYCheckoutOperation ()
 
+@property (strong, nonatomic, readonly) BUYClient *client;
```

Comment:
If the client goes away, this operation should consider itself cancelled. If the client goes away, we can be certain that there are no listeners who care about the results of the work done by this class or its inherent operations, so it should stop working and discard its results.